### PR TITLE
feat/KuCoin connector part 3: Market

### PR DESCRIPTION
**Pull request by vic-en**
Market files.

### CODE REVIEW

File: hummingbot/market/kucoin/kucoin_active_order_tracker.pyx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    def volume_for_bid_price(self, price) -> float:
+        return sum([float(msg["remaining_size"]) for msg in self._active_bids[price].values()])
+
+    def get_rates_and_quantities(self, entry) -> tuple:
+        return float(entry[0]), float(entry[1])
+
+    cdef tuple c_convert_diff_message_to_np_arrays(self, object message):
+        cdef:
+            dict content = message.content
+            list bid_entries = content["asks"]
```
Comment (erkmos) [inline hummingbot/market/kucoin/kucoin_active_order_tracker.pyx]: bids and asks are flipped here

Comment (ladeleke295) [reply hummingbot/market/kucoin/kucoin_active_order_tracker.pyx]: Corrected!

Comment (vic-en) [reply hummingbot/market/kucoin/kucoin_active_order_tracker.pyx]: corrected!

**Comment by CrimsonJacket** [comment]
test this please

**Comment by CrimsonJacket** [comment]
test this please

**Comment by CrimsonJacket** [comment]
test this please

**Pull request merged**